app/src/main/java/com/mobile/tandil/kotlinbaseproject/mvvm/model/Model.kt

```diff
@@ -0,0 +1,3 @@
+package com.mobile.tandil.kotlinbaseproject.mvvm.model
+
+class Model
```

Comment:
add a TODO comment explaining why this class is empty

Reply:
done